### PR TITLE
85 ramowka padding fix

### PR DESCRIPTION
**Pull request by Kszesiek**
Unified padding on the main page

obraz

### CODE REVIEW

File: mobile/lib/pages/recordings_page.dart [hunk 1]
```diff
@@ -36,26 +36,57 @@ class RecordingsPage extends HookWidget {
...
-          Duration(seconds: jsonData['media_details']['length'] as int)
-        ),
-      );
-      final thumbnailPath = element.thumbnailPath.isEmpty ? '' : await fetchSingle(
-        _singleRecordingUrl(element.thumbnailPath),
-            (jsonData) => jsonData['media_details']['sizes']['thumbnail']
-        ['source_url'] as String,
-      );
 
+    final Map<String, RecordingDetails> recordingDetails = {
```
Comment (julianuziemblo) [inline mobile/lib/pages/recordings_page.dart:40]: Dart linter is screaming that the explicit type declaration of local variable is not needed, so either remove it or, if you think it's neccessery (i don't think so) add a comment/a line in `analysis_options.yaml` disabling this lint

File: mobile/lib/pages/recordings_page.dart [hunk 2]
```diff
@@ -36,26 +36,57 @@ class RecordingsPage extends HookWidget {
...
+      for (final RecordingInfo recording in recordings)
+        recording.title: RecordingDetails()
```
Comment (julianuziemblo) [inline mobile/lib/pages/recordings_page.dart:42]: dart linter: missing trailing comma

File: mobile/lib/pages/recordings_page.dart [hunk 3]
```diff
@@ -36,26 +36,57 @@ class RecordingsPage extends HookWidget {
...
+    };
+    // using title as key isn't ideal to put it mildly, but out of all the
+    // fields in RecordingInfo I reckon this one is the least likely to be empty
+
+    final Iterable<Future<void>> recordingFutures = recordings.map((element) {
```
Comment (julianuziemblo) [inline mobile/lib/pages/recordings_page.dart:47]: dart linter: local variable explicit type declaration not needed

File: mobile/lib/pages/recordings_page.dart [hunk 4]
```diff
@@ -36,26 +36,57 @@ class RecordingsPage extends HookWidget {
...
+          ),
+        );
+        recordingDetails[element.title]?.recordingPath = recordingPath;
+        recordingDetails[element.title]?.duration = duration;
+      }
+
+      return fetchRecordingPathAndDuration();
+    });
+
+    final Iterable<Future<void>> thumbnailFutures = recordings.map((element) {
```
Comment (julianuziemblo) [inline mobile/lib/pages/recordings_page.dart:63]: dart linter: local variable explicit type declaration not needed

File: mobile/lib/pages/recordings_page.dart [hunk 5]
```diff
@@ -36,26 +36,57 @@ class RecordingsPage extends HookWidget {
...
+            : await fetchSingle(
+                _singleRecordingUrl(element.thumbnailPath),
+                (jsonData) => jsonData['media_details']['sizes']['thumbnail']
+                    ['source_url'] as String,
+              );
+        recordingDetails[element.title]?.thumbnailPath = thumbnailPath;
+      }
+
+      return fetchThumbnail();
+    });
```
Comment (julianuziemblo) [inline mobile/lib/pages/recordings_page.dart:76]: dart linter: missing trailing comma

**Review by julianuziemblo — CHANGES_REQUESTED** [review]
1 UI bug + some code formatting changes needed, overall this does great things for us!

Also: shouldn't this be 3, or at least 2 PRs? now the commit history will have a commit "85 ramowka padding fix" with... changes to recordings fetching? Let me know what you think

**Comment by Kszesiek** [comment]
> shouldn't this be 3, or at least 2 PRs? now the commit history will have a commit "85 ramowka padding fix" with... changes to recordings fetching? Let me know what you think

Yeah, definitely. I looked at the code, thought "I know a way to handle it better", decided to fiddle around with it a little, and... Here we are 😅 I take the blame here, my bad

File: mobile/lib/components/ra_list_widget.dart [hunk 6]
```diff
@@ -71,7 +71,7 @@ class _RaListWidgetItem extends StatelessWidget {
       height: rowHeight,
       color: color,
       child: Padding(
-        padding: const EdgeInsets.symmetric(horizontal: 3, vertical: 3),
+        padding: const EdgeInsets.symmetric(horizontal: 4, vertical: 4),
```
[the remark this answers is not in the document]
Comment (Kszesiek) [reply mobile/lib/components/ra_list_widget.dart:74]: Wow, I totally missed this one. Great eye 👀

**Comment by julianuziemblo** [comment]
I am also guilty of this, but if you could, please submit those changes in separate PR

... and if you see me do the same in the future, please remind me of this comment 😅

File: mobile/lib/components/ra_list_widget.dart (hunk 6 above)
[the remark this answers is not in the document]
Comment (Kszesiek) [reply mobile/lib/components/ra_list_widget.dart:74]: Also, that's weird, on my phone the letters aren't cut on the bottom like in the screenshot above

**Review by julianuziemblo — APPROVED** [review]
LGTM!

**Pull request merged**